course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- I originally split these topics into lecture and tutorial, but I will just call it 1 and 2 instead, since both of them will hopefully be in a computer lab. The third item under each week is for suggested homework.
+- Do we want simple quizzes at the start of each lecture, for the first ~5 weeks? This could serve as an easy to grade item and make sure that the students put effort in at home. It will make it easier for the students to know where they are at the drop date, and we can then reduce the quiz frequency as the course moves into the project based phase.
+
+### Week 1
+1. Introduction to the course, intro to the ecology part, motivation behind more reproducible workflows, talk briefly about spreadsheet and get straight into how to use R.
+    - Since Martin mentioned we should expect no programming experience from these students, I want to introduce programming at the first lecture already, at least conceptually.
+    - We can do this based on a [jupyter notebook I wrote previously](https://github.com/joelostblom/resources/blob/master/from-spreadsheets-to-pandas/from-spreadsheets-to-pandas.ipynb) with analogies between spreadsheet software and programming languages. It's in Python, but if you think this is a good idea, we can adopt it for R. If not, we can do something else.
+2. Basic R programming, vectors, data frames, loops, conditions, etc.
+3. Before 2nd class: Everyone have Rstudio up and running. Maybe we can also link them to some R intro course and ask them to complete 1 or 2 lectures there per week, which will serve as the underlying material for the quiz? This one is good imo http://tryr.codeschool.com/. Watch Christie's talk https://www.youtube.com/watch?v=TthxI9mlvGE?
+
+### Week 2
+1. Basic R-programming, repetition and continued, scripting.
+2. dplyr intro, finalize R-prep by teaching outstanding items required to use R for the following ecology classes.
+3. Again, link online lectures?
+
+### Week 3
+1. Ecology background knowledge for lab 1
+2. Ecology lab 1 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 4
+1. Ecology background knowledge for lab 2
+2. Ecology lab 2 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 5
+1. Introduce data sets, divide intro groups, think about interesting questions to ask, formulate hypothesis/plan of action.
+    - Potentially do this already the week before and have them take a survey with the data they want to work on and their background experience.
+2. Project work, getting started with their own data in R, tidy data principles, how to clean data, setting up a collaborative git environment with Rstudio (how much do we want to get into git?).
```

Comment:
If we use a streamlined data set for the beginning, it will be tidied already and most of the time will be spent on learning how to operate on that data set. Here, they will work with their actual data set, which most likely will need some tidying.